qemu/tests/nic_opt.py

```diff
@@ -0,0 +1,256 @@
+import os
+import logging
+import time
+
+from virttest import error_context
+from virttest import utils_net
+from virttest import utils_netperf
+from virttest import utils_misc
+from virttest import utils_test
+from virttest import data_dir
+from avocado.utils import cpu
+@error_context.context_aware
+def run(test, params, env):
+    """
+    NIC option check test.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def test_netperf():
+        """
+        Netperf stress test for nic option.
+        """
+        netperf_client = params.get("netperf_client").split()
+        netperf_server = params.get("netperf_server").split()
+        error_context.context("Run netperf stress test, client: %s, server: %s" %
+                              (netperf_client, netperf_server), logging.info)
+        guest_username = params.get("username", "")
+        guest_password = params.get("password", "")
+        host_password = params.get("password", "")
+        shell_client = params.get("shell_client")
+        shell_port = params.get("shell_port")
+        os_type = params.get("os_type")
+        shell_prompt = params.get("shell_prompt", r"^root@.*[\#\$]\s*$|#")
+        linesep = params.get(
+            "shell_linesep", "\n").encode().decode('unicode_escape')
+        status_test_command = params.get("status_test_command", "echo $?")
+        compile_option_client = params.get("compile_option_client", "")
+        compile_option_server = params.get("compile_option_server", "")
+        host_ip = utils_net.get_host_ip_address(params)
+        vms = params.get("main_vm")
+        server_infos = []
+        for server in netperf_server:
```

Comment:
since there is only one server, and the server is `localhost`, the loop and check could be removed.